app/views/proprietor/accounts/edit.html.erb

```diff
@@ -34,6 +34,32 @@
             <%= s.input :base_path %>
           <% end %>
 
+          <p>
+						<%= f.check_box :shared_search, {checked: @account.is_shared_search_enabled?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, 'true' , 'false' %>
```

Comment:
this needs some spacing adjustments please.

Reply:
I have actually moved this to a partial that i can shared between new and edit files